scipy/stats/_axis_nan_policy.py

```diff
@@ -226,6 +226,25 @@ def _check_empty_inputs(samples, axis):
     return output
 
 
+def _add_reduced_axes(res, reduced_axes, n_outputs,
+                      result_object):
+    res_expanded = []
+    if n_outputs == 1:
+        # Since NumPy 1.18, `expand_dims` supports tuple axis. As SciPy
+        # supports any NumPy >= 1.17, we can't guarantee support for tuple
+        # axis by NumPy. This for loop can be removed when support for
+        # NumPy 1.17 is dropped.
+        for a in reduced_axes:
+            res = np.expand_dims(res, axis=a)
+        res_expanded.append(res)
+    else:
+        for r in res:
+            for a in reduced_axes:
+                r = np.expand_dims(r, axis=a)
```

Comment:
This needs the same treatment for tuple axes as above. How about a separate `_expand_dims` function?

Reply:
Done.